src/System.Management.Automation/FormatAndOutput/common/TableWriter.cs

```diff
@@ -574,7 +586,8 @@ private static string GenerateRowField(string val, int width, int alignment, Dis
             return s;
         }
 
-        private const string ellipsis = "...";
+        private const char ellipsis = '\u2026';
```

Comment:
Is there any other scenarios, apart from `Format-Table`, where we truncate the output?

Reply:
A search for "..." across the code base found other hits, will address them in this PR.